zigate/protocol/response.py

```diff
@@ -144,6 +144,21 @@ def __str__(self):
 
 
 
+@register(0x8102)
+class IndividualAttributeReport(Response):
+
+    def __init__(self, *args):
+        super().__init__(*args)
+
+        self.struct = struct.Struct('!BHBHHBBH')
+        self.seq_nr, self.src_addr, self.endpoint, self.cluster_id, self.attr_enum, self.attr_status, self.atrr_data_type, self.attr_size = self.struct.unpack(self.value[:self.struct.size])
+        logger.debug("%s %s" % (self.struct.size, len(self.value[self.struct.size:])))
```

Comment:
The logger instance does it's own formatting which ignores errors: `logger.debug("%s %s", self.struct.size, len(self.value[self.struct.size:]))`
`